app/flatpak-builtins-utils.c

```diff
@@ -391,6 +391,14 @@ flatpak_resolve_matching_refs (const char *remote_name,
                                GError    **error)
 {
   guint chosen = 0;
+  const char *on = "";
+  const char *off = "";
+
+  if (flatpak_fancy_output ())
+    {
+      on = FLATPAK_ANSI_BOLD_ON;
```

Comment:
Bikeshed, but maybe `bold_start` and `bold_end` is more clear?

Reply:
Yeah `bold_on`/`bond_off` does seem better. Presumably should also change it in the existing instances in the codebase.

Reply:
Also we probably want to have both the single quotes and the bold, so the app/remote/etc strings have some delineation around them when they show up in log files.